plugins/woocommerce/src/Internal/Admin/Settings/PaymentsRestController.php

```diff
@@ -0,0 +1,1094 @@
+				'readonly'    => true,
+				'items'       => $this->get_schema_for_suggestion(),
+			),
+			'suggestion_categories'   => array(
+				'type'        => 'array',
+				'description' => esc_html__( 'The suggestion categories.', 'woocommerce' ),
+				'context'     => array( 'view', 'edit' ),
+				'readonly'    => true,
+				'items'       => array(
+					'type'        => 'object',
+					'description' => esc_html__( 'A suggestion category.', 'woocommerce' ),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+					'properties'  => array(
+						'id'          => array(
+							'type'        => 'string',
+							'description' => esc_html__( 'The unique identifier for the category.', 'woocommerce' ),
+							'context'     => array( 'view', 'edit' ),
+							'readonly'    => true,
+						),
+						'_priority'   => array(
+							'type'        => 'integer',
+							'description' => esc_html__( 'The priority of the category.', 'woocommerce' ),
+							'context'     => array( 'view', 'edit' ),
+							'readonly'    => true,
+						),
+						'title'       => array(
+							'type'        => 'string',
+							'description' => esc_html__( 'The title of the category.', 'woocommerce' ),
+							'context'     => array( 'view', 'edit' ),
+							'readonly'    => true,
+						),
+						'description' => array(
+							'type'        => 'string',
+							'description' => esc_html__( 'The description of the category.', 'woocommerce' ),
+							'context'     => array( 'view', 'edit' ),
+							'readonly'    => true,
+						),
+
+					),
+				),
+			),
+		);
+
+		return $schema;
+	}
+
+	/**
+	 * Get the schema for a payment gateway.
+	 *
+	 * @return array The schema for a payment gateway.
+	 */
+	private function get_schema_for_payment_gateway(): array {
+		return array(
+			'type'        => 'object',
+			'description' => esc_html__( 'A payment gateway.', 'woocommerce' ),
+			'properties'  => array(
+				'id'                => array(
+					'type'        => 'string',
+					'description' => esc_html__( 'The unique identifier for the payment gateway.', 'woocommerce' ),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'_order'            => array(
+					'type'        => 'integer',
+					'description' => esc_html__( 'The sort order of the payment gateway.', 'woocommerce' ),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'title'             => array(
+					'type'        => 'string',
+					'description' => esc_html__( 'The title of the payment gateway.', 'woocommerce' ),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'description'       => array(
+					'type'        => 'string',
+					'description' => esc_html__( 'The description of the payment gateway.', 'woocommerce' ),
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'short_description' => array(
```

Comment:
gateways don't contain short desc, we could remove it from schema

Reply:
But they might at some point. I am trying to make the gateways and suggestions more aligned.